### PR TITLE
I added Setup.md file

### PR DESCRIPTION
**Pull request by Nodirjn**
all the project was added

### CODE REVIEW

File: SETUP.md [hunk 1]
```diff
@@ -0,0 +1 @@
+I wanna learn c++ with my bro
```
Comment (farruhha) [inline SETUP.md]: Please add the following description "This is a mentorship program to teach C++ for everyone else."

**Review by farruhha — CHANGES_REQUESTED** [review]
@Nodirjn, I've reviewed your changes and left some feedback. Please fix it!

File: SETUP.md (hunk 1 above)
Comment (Nodirjn) [reply SETUP.md]: all was done

**Review by farruhha — APPROVED** [review]
It looks good to me, you can merge your code into main branch.

**Pull request merged**